CHANGELOG.md

```diff
@@ -2,10 +2,13 @@
 All notable changes to this project will be documented in this file.
 This project aims to adhere to [Semantic Versioning](http://semver.org/).
 
+## [3.2.6] -
```

Comment:
I know we don't have a crisp story in #316.  Are the method signature changes made here okay to do in a patch version?